cortex/utils.py

```diff
@@ -141,24 +142,24 @@ def verify_JWT(token, config=None):
         return generate_token(config)
 
 
-def _get_fabric_info(config: dict):
+def _get_fabric_info(config: dict, verify_ssl_cert: Union[bool, str]=True):
     uri = config.get("url") + "/fabric/v4/info"
     headers = {"Content-Type": "application/json"}
-    return request("GET", uri, headers=headers).json()
+    return request("GET", uri, headers=headers, verify=verify_ssl_cert,).json()
 
 
-def _get_fabric_server_ts(config: dict):
-    return _get_fabric_info(config).get("serverTs")
+def _get_fabric_server_ts(config: dict, verify_ssl_cert: Union[bool, str]=True):
+    return _get_fabric_info(config, verify_ssl_cert).get("serverTs")
 
 
-def generate_token(config, validity=2):
+def generate_token(config, verify_ssl_cert: Union[bool, str]=True, validity=2):
```

Comment:
Might also be useful to add the parameters to the docstring, so you can mention that `verify_ssl_cert` is either a bool or the path to CA bundle